lib/app_manifest/environment.rb

```diff
@@ -0,0 +1,21 @@
+module AppManifest
+  class Environment
+    include Virtus.model
+    include Serializer
+
+    attribute :addons,      Array[Addon]
```

Comment:
Good idea, I prefer composition over the inheritance method I used here, anyway.